src/BlobBasics.java

```diff
@@ -122,31 +108,29 @@ else if (s.getErrorCode().equals("FeatureNotSupportedByEmulator")) {
 
             // Enumerate all containers starting with the prefix "blobbasics-" and list all blobs
             System.out.println("\nEnumerate all containers and starting with the prefix \"blobbasics-\" list all blobs");
-            for (CloudBlobContainer container : blobClient.listContainers("blobbasics-")) {
-                System.out.println(String.format("\tContainer: %s", container.getName()));
-                for (ListBlobItem blob : container.listBlobs()) {
-                    if (blob instanceof CloudBlob) {
-                        System.out.println(String.format("\t\t%s\t: %s", ((CloudBlob) blob).getProperties().getBlobType(), blob.getUri().toString()));
-                    }
-                }
-            }
+            blobServiceClient.listBlobContainers().stream().filter(blobContainerItem -> blobContainerItem.getName().startsWith("blobbasics-")).forEach(blobContainerItem -> {
```

Comment:
see earlier comment about using other listBlobContainers method

Reply:
The code has been fixed according to your suggestion